framework/components/ecl-page-headers/ecl-page-headers.twig

```diff
@@ -0,0 +1,48 @@
+{% if variant is defined and variant == "highlight" %}
+<div class="ecl-page-header ecl-page-header--highlight">
+  <div class="ecl-page-header__body">
+    <div class="ecl-page-header__label">{{ label }}</div>
+    <div class="ecl-page-header__highlight">
+      {{ highlight }}
+    </div>
+  </div>
+</div>
+{% else %}
+<div class="ecl-page-header">
+  {% include '@ec-europa/ecl-breadcrumbs' with { 'extraClass':'ecl-u-mh-xxs','segments': [{'target':'#','title':'European Commission'},{'target':'#','title':'Announcements'}] } %}
+
+  <div class="ecl-page-header__body">
+
+    {% if identity is defined %}
+    <div class="ecl-page-header__identity">
+      {{ identity }}
+    </div>
+    {% endif %}
+
+    {% if meta is defined %}
+    <div class="ecl-page-header__meta">
+      <div class="ecl-meta ecl-meta--header">
```

Comment:
I think we can create a follow-up ticket to improve the `@ec-europa/ecl-meta` to be to use it here as:

```
      {% include '@ec-europa/ecl-meta' with {
        'modifier': 'ecl-meta--header',
        'meta_item_type_data': meta.type,
        'meta_item_data': '....',
      } %}
```

However, the component is not ready to have multiple `ecl-meta__item` and cannot accommodate `date-display-single` at the moment, which is definitely out of scope.